fix splunk max buffer size (#227)
centreon/centreon-stream-connector-scripts: modules/docs/README.md

| boolean_to_number                  | change a true/false boolean to a 1/0 value                                                                      | [Documentation](sc_common.md#boolean_to_number-method)                  |
| number_to_boolean                  | change a 0/1 number to a false/true value                                                                       | [Documentation](sc_common.md#number_to_boolean-method)                  |
| check_boolean_number_option_syntax | make sure that a boolean is 0 or 1, if that's not the case, replace it with a default value                     | [Documentation](sc_common.md#check_boolean_number_option_syntax-method) |
| split                              | split a string using a separator (default is ",") and store each part in a table                                | [Documentation](sc_common.md#split-method)                              |
| compare_numbers                    | compare two numbers using the given mathematical operator and return true or false                              | [Documentation](sc_common.md#compare_numbers-method)                    |

> | generate_postfield_param_string    | convert a table of parameters into an url encoded parameters string                                             | [Documentation](sc_common.md#generate_postfield_param_string-method)    |

Reviewer: ```suggestion
| generate_postfield_param_string    | convert a table of parameters into a URL encoded parameter string                                             | [Documentation](sc_common.md#generate_postfield_param_string-method)    |
```

Reviewer: Is "a URL encoded parameter string" a thing?

> | load_json_file                     | method loads a json file and parse it                                                                           | [Documentation](sc_common.md#load_json_file-method)                     |

Reviewer: ```suggestion
| load_json_file                     | the method loads a json file and parses it                                                                           | [Documentation](sc_common.md#load_json_file-method)                     |
```

| json_escape                        | escape json characters in a string                                                                              | [Documentation](sc_common.md#json_escape-method)                        |
| xml_escape                         | escape xml characters in a string                                                                               | [Documentation](sc_common.md#xml_escape-method)                         |
| lua_regex_escape                   | escape lua regex special characters in a string                                                                 | [Documentation](sc_common.md#lua_regex_escape-method)                   |

> | dumper                             | dump any variable for debug purpose                                                                             | [Documentation](sc_common.md#dumper-method)                             |

Reviewer: ```suggestion
| dumper                             | dump any variable for debug purposes                                                                             | [Documentation](sc_common.md#dumper-method)                             |
```

| trim                               | trim spaces (or provided character) at the beginning and the end of a string                                    | [Documentation](sc_common.md#trim-method)                               |
| get_bbdo_version                   | returns the first digit of the bbdo protocol version                                                            | [Documentation](sc_common.md#get_bbdo_version-method)                   |
| is_valid_pattern                   | check if a Lua pattern is valid                                                                                 | [Documentation](sc_common.md#is_valid_pattern-method)                   |
| sleep                              | wait a given number of seconds                                                                                  | [Documentation](sc_common.md#sleep-method)                              |

> | create_sleep_counter_table         | create a table to handle sleep counters. Useful when you want to log something less often after some repetition | [Documentation](sc_common.md#create_sleep_counter_table-method)         |

Reviewer: ```suggestion
| create_sleep_counter_table         | create a table to handle sleep counters. Useful when you want to log something less often after some repetitions | [Documentation](sc_common.md#create_sleep_counter_table-method)         |
```

## sc_param methods

| Method name                        | Method description                                                                                                                            | Link                                                                   |
| ---------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------- |
| param_override                     | replace default values of params with the ones provided by users in the web configuration of the stream connector                             | [Documentation](sc_param.md#param_override-method)                     |

> | check_params                       | make sure that the default stream connectors params provided by the user from the web configuration are valid. If not, uses the default value | [Documentation](sc_param.md#check_params-method)                       |

Reviewer: ```suggestion
| check_params                       | make sure that the default stream connector params provided by the user from the web configuration are valid. If not, uses the default value | [Documentation](sc_param.md#check_params-method)                       |
```

| is_mandatory_config_set            | check that all mandatory parameters for a stream connector are set                                                                            | [Documentation](sc_param.md#is_mandatory_config_set-method)            |
| get_kafka_params                   | retreive Kafka dedicated parameters from the parameter list and put them in the provided kafka_config object                                  | [Documentation](sc_param.md#get_kafka_params-method)                   |
| load_event_format_file             | load a file that serves as a template for formatting events                                                                                   | [Documentation](sc_param.md#load_event_format_file-method)             |

> | build_accepted_elements_info       | build a table that store information about accepted elements                                                                                  | [Documentation](sc_param.md#build_accepted_elements_info-method)       |

Reviewer: ```suggestion
| build_accepted_elements_info       | build a table that store information about accepted elements                                                                                  | [Documentation](sc_param.md#build_accepted_elements_info-method)       |
```
